outputs.tf

```diff
@@ -11,11 +11,11 @@ output "vpc_id" {
 }
 
 output "public_route_table_id" {
-  value = "${aws_route_table.public.id}"
+  value = "${join(",", aws_route_table.public.*.id)}"
 }
 
 output "private_route_table_id" {
-  value = "${aws_route_table.private.id}"
+  value = "${join(",", aws_route_table.private.*.id)}"
```

Comment:
I think we should return native lists (as on line 2,6), instead of using join/split.